polylogue/core/config_validation.py

```
@@ -63,28 +37,41 @@ class ExportsModel(BaseModel):
-    drive: Optional[DriveModel] = None
+    paths: Optional[PathsModel] = None
+    ui: Optional[UiModel] = None
+    defaults: Optional[UiModel] = None
     index: Optional[IndexModel] = None
-    exports: Optional[ExportsModel] = None
-    defaults: Optional[DefaultsModel] = None
 
     model_config = ConfigDict(extra="forbid")
 
     @model_validator(mode="after")
     def ensure_theme(cls, values: "ConfigModel") -> "ConfigModel":
```

Ruff (0.15.9)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)